store/conf/attrs/zimbra-attrs.xml

```diff
@@ -10391,4 +10391,37 @@ TODO: delete them permanently from here
   <desc>Whether retention policy feature is enabled</desc>
 </attr>
 
+<attr id="4102" name="zimbraTwoFactorAuthMethodAllowed" type="string" cardinality="multi" optionalIn="domain,cos,account" flags="accountCosDomainInherited,accountInfo" since="11.0.0">
```

Comment:
looks like since value needs to be changed to 10.1.0 for all new attributes for classic, please check other attrs added for reference

Reply:
its fine, it is 11.0.0